OpenOversight/app/models/database.py

```diff
@@ -76,7 +80,7 @@ class Job(BaseModel):
     )
 
     def __repr__(self):
-        return "<Job ID {}: {}>".format(self.id, self.job_title)
+        return f"<Job ID {self.id,}: {self.job_title}>"
```

Comment:
I think there is a script we can run over the codebase to update all .format to f-strings

Reply:
It looks like we can use this: https://stackoverflow.com/a/56877138

I'll make a PR that runs it on the codebase. If it's useful beyond that, I may throw it in pre-commit.

Reply:
Taken care of here: https://github.com/lucyparsons/OpenOversight/pull/991